Index: /src/pages/exercises/[slug].astro
===================================================================
--- /src/pages/exercises/[slug].astro
+++ /src/pages/exercises/[slug].astro
@@ -1,5 +1,9 @@
 ---
+import { render } from 'astro:content';
+import { getCollection } from 'astro:content';
+
 export async function getStaticPaths() {
   // This expects us to returns a collection of items to render
 }
 ---
+\
\ No newline at end of file
