Index: /src/components/BlockList.astro
===================================================================
--- /src/components/BlockList.astro
+++ /src/components/BlockList.astro
@@ -8,7 +8,13 @@
   blocks: ApiBlockListModelItemsItem[];
 }
 
 const { blocks } = Astro.props;
+
+const COMPONENT_MAP = {
+  richTextComponent: RichTextComponent,
+  imageComponent: ImageComponent,
+  quoteComponent: QuoteComponent,
+};
 ---
 
 <!-- Render Blocks -->
