Index: /src/pages/blog/[slug].astro
===================================================================
--- /src/pages/blog/[slug].astro
+++ /src/pages/blog/[slug].astro
@@ -5,11 +5,12 @@
   );
 
   let blogs = await data.json();
 
-  // Return the collection in a format that Astro expects
-  // -- Set the URL/Path
-  // -- Pass data to render
+  return blogs.items.map((article: any) => ({
+    params: { slug: article.route.path.replace('/blog/', '') },
+    props: { article },
+  }));
 }
 ---
 
 <!-- Render Blog content here  -->
