Index: /src/pages/blog/[slug].astro
===================================================================
--- /src/pages/blog/[slug].astro
+++ /src/pages/blog/[slug].astro
@@ -10,7 +10,10 @@
     params: { slug: article.route.path.replace('/blog/', '') },
     props: { article },
   }));
 }
+
+const { article } = Astro.props;
 ---
 
-<!-- Render Blog content here  -->
+<h1>{article.properties?.title}</h1>
+<div set:html={article.properties.richText.markup} />
