Index: /src/content.config.ts
===================================================================
--- /src/content.config.ts
+++ /src/content.config.ts
@@ -1,6 +1,7 @@
 import { glob } from 'astro/loaders';
 import { defineCollection, z } from 'astro:content';
+import { getAllContentByType } from './services/ContentService';
 
 const workshopExercise = defineCollection({
   loader: glob({ pattern: '**/*.md', base: './src/data/workshop/exercises' }),
   schema: z.object({
@@ -8,5 +9,12 @@
     description: z.string(),
   }),
 });
 
+const blog = defineCollection({
+  loader: async () => {
+    const response = await getAllContentByType('blogArticlePage');
+    // Return articles
+  },
+});
+
 export const collections = { workshopExercise };
