Index: /src/content.config.ts
===================================================================
--- /src/content.config.ts
+++ /src/content.config.ts
@@ -12,9 +12,13 @@
 
 const blog = defineCollection({
   loader: async () => {
     const response = await getAllContentByType('blogArticlePage');
-    // Return articles
+    return response.map((article) => ({
+      id: article.id,
+      slug: article.route.path,
+      content: article,
+    }));
   },
 });
 
-export const collections = { workshopExercise };
+export const collections = { workshopExercise, blog };
