Index: /src/pages/blog/index.astro
===================================================================
--- /src/pages/blog/index.astro
+++ /src/pages/blog/index.astro
@@ -1,12 +1,8 @@
 ---
 import Layout from '../../layouts/Layout.astro';
-
-let data = await fetch(
-  'http://localhost:48006/umbraco/delivery/api/v2/content?take=100&filter=contentType:blogArticlePage'
-);
-
-let blogs = await data.json();
+import { getCollection } from 'astro:content';
+const blogs = await getCollection('blog');
 ---
 
 <Layout>
   <div class="bg-white py-10">
@@ -16,11 +12,11 @@
           Blog Listing
         </h2>
         <div class="p-4">
           {
-            blogs.items.map((article: any) => (
+            blogs.map((article) => (
               <article>
-                <a href={`${article.route.path}`}>{article.name}</a>
+                <a href={`${article.data.slug}`}>{article.data.content.name}</a>
               </article>
             ))
           }
         </div>
